locale/en.yml

```diff
@@ -103,40 +103,99 @@ en:
         v_derived_cpu_reserved_pct:    CPU - Reserved (%)
         v_derived_memory_reserved_pct: Memory - Reserved (%)
 
+      abs_max_cpu_usagemhz_rate_average_timestamp: Absolute Max CPU Usage MHz Rate Average Timestamp
+      abs_max_cpu_usagemhz_rate_average_value: Absolute Max CPU Usage MHz Rate Average Value
       abs_max_cpu_usage_rate_average_timestamp: CPU - Absolute Max Usage Rate (Timestamp)
       abs_max_cpu_usage_rate_average_value: CPU - Absolute Max Usage Rate (%)
+      abs_max_derived_memory_used_timestamp: Absolute Max Derived Memory Used Timestamp
+      abs_max_derived_memory_used_value: Absolute Max Derived Memory Used Value
       abs_max_disk_usage_rate_average_timestamp: Disk I/O - Absolute Max Usage Rate (Timestamp)
       abs_max_disk_usage_rate_average_value: Disk I/O - Absolute Max Usage Rate (KBps)
       abs_max_mem_usage_absolute_average_timestamp: Memory - Absolute Max Usage Rate (Timestamp)
       abs_max_mem_usage_absolute_average_value: Memory - Absolute Max Usage Rate (%)
       abs_max_net_usage_rate_average_timestamp: Network I/O - Absolute Max Usage Rate (Timestamp)
       abs_max_net_usage_rate_average_value: Network I/O - Absolute Max Usage Rate (KBps)
+      abs_min_cpu_usagemhz_rate_average_timestamp: Absolute Min CPU Usage MHz Rate Average Timestamp
+      abs_min_cpu_usagemhz_rate_average_value: Absolute Min CPU Usage MHz Rate Average Value
       abs_min_cpu_usage_rate_average_timestamp: CPU - Absolute Min Usage Rate (Timestamp)
       abs_min_cpu_usage_rate_average_value: CPU - Absolute Min Usage Rate (%)
+      abs_min_derived_memory_used_timestamp: Absolute Min Derived Memory Used Timestamp
+      abs_min_derived_memory_used_value: Absolute Min Derived Memory Used Value
       abs_min_disk_usage_rate_average_timestamp: Disk I/O - Absolute Min Usage Rate (Timestamp)
       abs_min_disk_usage_rate_average_value: Disk I/O - Absolute Min Usage Rate (KBps)
       abs_min_mem_usage_absolute_average_timestamp: Memory - Absolute Min Usage Rate (Timestamp)
       abs_min_mem_usage_absolute_average_value: Memory - Absolute Min Usage Rate (%)
       abs_min_net_usage_rate_average_timestamp: Network I/O - Absolute Min Usage Rate (Timestamp)
       abs_min_net_usage_rate_average_value: Network I/O - Absolute Min Usage Rate (KBps)
+      access_modes: Access Modes
```

Comment:
Any of the ones where the value can be inferred from the original key I thought should not be here. For example `access_modes` automatically implies `Access Modes`  via the `titleize` method.  It's possible we need to update the i18n generator to deal with other keys.  @mzazrivec ?

Reply:
As far as I can tell, all the strings that are being added in this PR were generated automatically from the dictionary key by `.titleize`. The results therefore don't always make sense, for example: `retirement_last_warn: Retirement Last Warn`

Also, I see that this PR updates model names, for example: `Network: Networks`: the model name should be in singular form.

I'm not entirely sure what you mean by i18n generator here. What we do with the `locale/en.yml` file is that we extract the pretty looking column / model / table names, wrap them in `_()` and put them into a temporary ruby file so that subsequent string extraction finds these strings. We don't really generate anything here (other than the temporary ruby file).

So if the string exists in the `locale/en.yml` file, it will make it into the dictionary, if it doesn't exist in the yaml file, it won't be translated. Seeing bunch of strings being added here makes me wonder where the strings (or in fact the string keys) come from. Did we really miss all these strings before?

Reply:
`Dictionary.gettext()` does the `.titleize()` thing here.

```
irb(main):003:0> Dictionary.gettext('non_sense', :type => :column)
=> "non_sense"
irb(main):004:0> Dictionary.gettext('non_sense', :type => :column, :notfound => :titleize)
=> "Non Sense"
```

Am I correct to understand that you'd rather see the gettext strings being generated automatically rather than maintaining this yaml file?